packages/polymath-shared/src/utils/format.js

```diff
@@ -87,3 +87,16 @@ export const toTokens = (
   const number = new BigNumber(value);
   return number.toFormat(decimals);
 };
+
+export const toLargeNumber = (
+  data: number,
+  precision: number,
+  decimals: number
+) => {
+  const str = data.toString();
```

Comment:
@sajclarke not sure what this method is doing. Can you explain a bit in more detail what it does?